data/bangs.json

```diff
@@ -109935,5 +109935,13 @@
     "u": "https://vibe.naver.com/search?query={{{s}}}",
     "c": "Multimedia",
     "sc": "Music"
+  },
```

Comment:
Weird request, but could you add this somewhere in the middle of the file? Adding bangs to the end tends to make merge conflicts which neex to be manually resolved